js/examples/openai/src/index.ts

```diff
@@ -12,13 +12,15 @@ const openai = new OpenAI({
     model: "gpt-3.5-turbo",
   });
   chatCompletion.choices.forEach((choice) => {
+    // eslint-disable-next-line no-console
     console.log(choice.message);
   });
   chatCompletion = await openai.chat.completions.create({
     messages: [{ role: "user", content: "Tell me a joke" }],
     model: "gpt-3.5-turbo",
   });
   chatCompletion.choices.forEach((choice) => {
+    // eslint-disable-next-line no-console
     console.log(choice.message);
```

Comment:
Assuming we want to nix these?

Reply:
No, it's okay - It's just that no console is good for the instrumentations but not the examples. We will figure out an override